### DISCUSSION TITLE
（灌水）索伯表示；我们将在下个赛季拿到积分

### DISCUSSION THREAD

**Discussion by WXZ1025** [share121/inter-knot#2800] [其他]
不是哥们你真信啊，不过我还是祝愿索伯下辈子努努力，争取拿到积分吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
![Image_1723977763313](https://github.com/user-attachments/assets/1a23dddd-f63d-4d58-ad46-db0ed5fdca0d)

**Comment by flemon-y** [comment]
其他车队：会赢吗？

**Comment by KEARUM** [comment]
什么科幻剧情
